src/content/nl/2019/accessibility.md

```diff
@@ -0,0 +1,320 @@
+
+{{ figure_markup(
+  image="fig2.png",
+  caption="Percentage sites dat zoomen en schalen uitschakelt versus apparaattype.",
+  description="Verticale meetpercentagegegevens, variërend van 0 tot 80 in stappen van 20, versus het apparaattype, gegroepeerd in desktop en mobiel. Desktop ingeschakeld: 75,46%; Desktop uitgeschakeld 24,54%; Mobiel ingeschakeld: 67,79%; Mobiel uitgeschakeld: 32,21%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=2053904956&format=interactive"
+  )
+}}
+
+### Taalidentificatie
+
+Het web staat vol met wonderbaarlijke hoeveelheden inhoud. Er is echter een addertje onder het gras: er bestaan meer dan 1000 verschillende talen in de wereld en de inhoud die u zoekt, is mogelijk niet geschreven in een taal die u vloeiend beheerst. In de afgelopen jaren hebben we grote vooruitgang geboekt op het gebied van vertaaltechnologieën en u heeft er waarschijnlijk een op internet gebruikt (bijv. Google translate).
+
+Om deze functie te vergemakkelijken, moeten de vertaalmachines weten in welke taal uw pagina's zijn geschreven. Dit wordt gedaan met behulp van het [`lang` attribuut](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang). Zonder dit moeten computers raden in welke taal uw pagina is geschreven. Zoals u zich wellicht kunt voorstellen, leidt dit tot veel fouten, vooral wanneer pagina's meerdere talen gebruiken (uw paginanavigatie is bijvoorbeeld in het Engels, maar de inhoud van het bericht is in het Japans).
+
+Dit probleem is zelfs nog meer uitgesproken bij tekst-naar-spraak-ondersteunende technologieën zoals schermlezers, waar, als er geen taal is gespecificeerd, ze de neiging hebben om de tekst in de standaard gebruikerstaal te lezen.
+
+Van de geanalyseerde pagina's specificeert 26,13% geen taal met het kenmerk `lang`. Hierdoor blijft meer dan een kwart van de pagina's vatbaar voor alle hierboven beschreven problemen. Het goede nieuws? Van de sites die het `lang`-attribuut gebruiken, specificeren ze 99,68% van de tijd een geldige taalcode correct.
+
+### Afleidende inhoud
+
+Sommige gebruikers, zoals mensen met cognitieve beperkingen, hebben moeite om zich gedurende lange tijd op dezelfde taak te concentreren. Deze gebruikers willen niet omgaan met pagina's met veel beweging en animaties, vooral als deze effecten puur cosmetisch zijn en geen verband houden met de taak die voorhanden is. Deze gebruikers hebben minimaal een manier nodig om alle afleidende animaties uit te schakelen.
+
+Helaas geven onze bevindingen aan dat oneindig herhalende animaties vrij gebruikelijk zijn op internet, waarbij 21,04% van de pagina's ze gebruikt via oneindige CSS-animaties of [`<marquee>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee) en [`<blink>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blink) elementen.
```

Comment:
Can you check all MDN references to see if Dutch versions exist and use them if they do?

```suggestion
Helaas geven onze bevindingen aan dat oneindig herhalende animaties vrij gebruikelijk zijn op internet, waarbij 21,04% van de pagina's ze gebruikt via oneindige CSS-animaties of [`<marquee>`](https://developer.mozilla.org/nl/docs/Web/HTML/Element/marquee) en [`<blink>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blink) elementen.
```

Reply:
Should I check all links for dutch versions?

Reply:
Ideally yes. Not the most exciting part of the job this - we should find some way to automate this!